ppr-ui/src/composables/common/useAuth.ts

```diff
@@ -0,0 +1,13 @@
+import { computed, ComputedRef } from 'vue-demi'
+import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
+
+export const useAuth = () => {
+  /** Helper to check if the current user is authenticated */
+  const isAuthenticated: ComputedRef<boolean> = computed((): boolean => {
+    return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
```

Comment:
It probably would. 
It works and is generally acceptable both ways.
`Boolean()` in this context is acceptable because it in reference to an expression. Where we want to be careful NOT to use Boolean is when defining primitive `boolean` types. 
😵 hahah i know right..